tests/e2e/initialization/node.go

```diff
@@ -310,6 +311,7 @@ func (n *internalNode) initValidatorConfigs(c *internalChain, persistentPeers []
 	valConfig.StateSync.Enable = false
 	valConfig.LogLevel = "info"
 	valConfig.P2P.PersistentPeers = strings.Join(persistentPeers, ",")
+	valConfig.Consensus.TimeoutCommit = 5 * time.Millisecond
```

Comment:
Note that for this change to take effect we need to backport this to `v10.x` and rebuild the chain-init image. Then, it has to be updated here:
https://github.com/osmosis-labs/osmosis/blob/8e6e4de0bc65d521185f2aa82ff5884c107bd8c5/tests/e2e/containers/config.go#L30

If the `initialization` package is updated, the `previousVersionInit` tag should be updated as well

This is a manual step to make sure that we are able to initialize genesis and configs of the previous Osmosis version correctly.